cocos/renderer/backend/opengl/CommandBufferGL.cpp

```diff
@@ -329,7 +329,7 @@ void CommandBufferGL::bindVertexBuffer(ProgramGL *program) const
     const auto& vertexLayouts = getVertexLayouts();
     for (const auto& vertexBuffer : _vertexBuffers)
     {
-        if (! vertexBuffer)
+        if (! vertexBuffer || attributeInfos.empty())
```

Comment:
`attributeInfos.empty()` can be checked before `for loop`.

Reply:
```suggestion
        if (! vertexBuffer)
```